### ISSUE TITLE
The screen stutters every five seconds while Redshift is running

### ISSUE THREAD

**Issue by maxigaz** [jonls/redshift#861]
**Describe the bug**
While Redshift is running in continuous mode, the screen stutters exactly every five seconds.

- [X] [I have checked the FAQ](https://github.com/jonls/redshift/blob/master/README.md#faq) and my issue is not mentioned there.

**To Reproduce**
1. Run `redshift` in the terminal.
2. Open a video with Firefox (e.g. YouTube) or mpv, or open Retroarch (load any core or just watch the menu animations with xmb menu driver selected).
3. The video stutters every five seconds.

**Expected behavior**
No stuttering.

**Software versions:**
 - OS: Arch Linux x64_64
 - GPU driver: radeon (open source driver for old ATI/AMD cards)
 - DE/WM: i3
 - Redshift version: 1.12
 - Redshift installed from: official repos with pacman

**Additional information**
Other than killing Redshift with `pkill redshift`, a possible workaround is running it in one-shot mode:

```
redshift -O 6500K -b 0.8:1
```

Also, someone has reported the same problem on the [Arch Linux Forums][1], and I couldn't find it here yet, so I posted it here.

[1]: https://bbs.archlinux.org/viewtopic.php?id=273933

**Comment by Augusto7743** [comment]
Try run the command below, but is only for test because will disable Redshift service.
systemctl mask --user redshift.service

If not fix reinstall Redshit to re-enable redshift.service.

**Comment by maxigaz** [comment]
@Augusto7743 So, `systemctl mask --user redshift.service` disables the service and makes it unusable until Redshift is reinstalled, doesn't it? Why is it needed?

When I noticed the issue, I didn't even use the service file; I launched `redshift` through i3 on startup. And the stuttering was only present if Redshift was running in the background, as I wrote in the title of my report.

Anyway, I can't reproduce the issue at the moment, so it looks like it has been fixed somewhere. I'm closing this for now.
